src/safeds/ml/classical/classification/_decision_tree_classifier.py

```diff
@@ -71,3 +73,36 @@ def _get_sklearn_model(self) -> ClassifierMixin:
             max_depth=self._max_depth,
             min_samples_leaf=self._min_sample_count_in_leaves,
         )
+
+    # ------------------------------------------------------------------------------------------------------------------
+    # Plot
+    # ------------------------------------------------------------------------------------------------------------------
+
+    def plot(self) -> Image:
+        """
+        Get the image of the decision tree.
+
+        Returns
+        -------
+        Image:
+            The decision tree figure as an image.
+        """
```

Comment:
Add `Raises` section to highlight that the method raises if the model is not fitted.

Reply:
Fixed